src/pyggdrasil/visualize/_mcmc.py

```diff
@@ -1,124 +1,122 @@
 """Methods for visualizing mcmc samples."""
-
+import matplotlib
 import matplotlib.pyplot as plt
 
-from numpy import ndarray
+import json
 from pathlib import Path
 
-from pyggdrasil import TreeNode
 from pyggdrasil.interface import PureMcmcData
-from pyggdrasil.distances import TreeSimilarityMeasure, calculate_distance_matrix
-from pyggdrasil.visualize import plot_tree_mcmc_sample, plot_tree_no_print
+from pyggdrasil.visualize import plot_tree_mcmc_sample
+
+# use SVG backend for matplotlib
+matplotlib.use("SVG")
 
```

Comment:
Choosing a non-interactive (allows for multitheading), system-independent backend for plot generation.